crates/sys/src/lib.rs

```diff
@@ -29,12 +29,18 @@ mod inlines;
 )]
 mod bindings;
 
-/// Create a static C string.
+/// Create a static C string of type [`&'static CStr`][`core::ffi::CStr`].
 /// Will automatically add a NUL terminator.
+pub use cstr::cstr;
```

Comment:
Note this addition in the changelog under the "Added" section:
```markdown
- `flipperzero_sys::cstr`
```

Reply:
Done